### PR TITLE
Use some API types from `janus_messages`

### LINKED ISSUE #88: Wire formats for HpkeConfig don't match, divviup-api uses numbers for algorithm IDs and an array of numbers for the public key, while Janus uses strings and a base64url-encoded string respectively.

### PR DESCRIPTION
**Pull request by jbr**
I believe this closes #88

### CODE REVIEW

File: src/clients/aggregator_client/api_types.rs [hunk 1]
```diff
@@ -118,53 +111,54 @@
...
-            task_expiration: value
+            task_expiration: new_task
                 .expiration
                 .map(|task| task.unix_timestamp().try_into().unwrap())
                 .unwrap_or(u64::MAX),
-            min_batch_size: value.min_batch_size.unwrap(),
-            time_precision: value.time_precision_seconds.unwrap(),
-            collector_hpke_config: value.hpke_config.unwrap().into(),
-        }
+            min_batch_size: new_task.min_batch_size.unwrap().try_into()?,
```
[the remark this answers is not in the document]
Comment (jbr) [reply src/clients/aggregator_client/api_types.rs:158]: who are you calling useless

File: Cargo.toml [hunk 2]
```diff
@@ -20,12 +20,14 @@ base64 = "0.21.0"
 env_logger = "0.10.0"
 fastrand = "1.9.0"
 git-version = "0.3.5"
+janus_messages = { git = "https://github.com/divviup/janus.git" }
```
Comment (jbr) [inline Cargo.toml:23]: We probably should add versioning of some sort to this eventually so changes on main don't break builds

Comment (divergentdave) [reply Cargo.toml:23]: This is published at https://crates.io/crates/janus_messages and there haven't been big changes in a while, we should use that.

Comment (jbr) [reply Cargo.toml]: resolved in 78e8cef41ad48b282a765bdaec479cfebdb6cf47

File: src/clients/aggregator_client/api_types.rs [hunk 3]
```diff
@@ -41,38 +46,26 @@ impl From<Vdaf> for VdafInstance {
...
-            id: value.id.unwrap(),
-            kem_id: value.kem_id.unwrap(),
-            kdf_id: value.kdf_id.unwrap(),
-            aead_id: value.aead_id.unwrap(),
-            public_key: value.public_key.unwrap().into_bytes(),
+            id: Some((*hpke_config.id()).into()),
+            kem_id: Some((*hpke_config.kem_id()) as u16),
+            kdf_id: Some((*hpke_config.kdf_id()) as u16),
+            aead_id: Some((*hpke_config.aead_id()) as u16),
+            public_key: Some(URL_SAFE_NO_PAD.encode(hpke_config.public_key())),
```
Comment (jbr) [inline src/clients/aggregator_client/api_types.rs:68]: This duplicates https://github.com/divviup/janus/pull/1356. It seems preferable for the string representation to live in janus_messages and to just call ToString::to_string here

**Pull request merged**